CustomErrors.py

```diff
@@ -154,6 +154,12 @@ class ICPFileNotNamedCorrectly(Error):
     def __init__(self, path):
         self.message = "ERROR: " + path + " was not uploaded to the database because the file was not properly named. The file must " \
                           "be named according to the following convention: year.project.operator.xlsx. Please rename the file and try again.\n\n"
+
+class NewSRPFileNotNamedCorrectly(Error):
+    def __init__(self, path):
+        self.message = "ERROR: " + path + " was not uploaded to the database because the file was not properly named. The file must " \
+                                          "be named according to the following convention: date.project.xlsx. Please rename the file and try again.\n\n"
```

Comment:
This was a grey area for me, since we're getting this from the EAL do we want people to rename the file based on the Date found in the first 10 or so fluff rows? Or should I make Datatiger automate pulling that from those rows regardless? (I wasn't sure which date values are important so as of now I'm only saving datetime_uploaded to the database)